lang/en/auth_userkey.php

```diff
@@ -38,4 +38,7 @@
 $string['redirecturl_desc'] = 'Optionally you can redirect users to this URL after they logged out from LMS.';
 $string['incorrectredirecturl'] = 'You should provide valid URL';
 $string['userkey:generatekey'] = 'Generate login user key';
-$string['pluginisdisabled'] = 'The userkey authentication plugin is disabled.';
\ No newline at end of file
+$string['pluginisdisabled'] = 'The userkey authentication plugin is disabled.';
+$string['ssourl'] = 'URL of SSO host';
+$string['ssourl_desc'] = 'URL of the SSO host to redirect users to. '.
+                         'If defined users will be redirected here on login instead of the Moodle Login page';
```

Comment:
there is a bug in AMOS where it can't correctly parse string like this, please make this a single line. Pretty sure the travis linter is aware of this and won't complain inside lang files